lib/seahorn/BvOpSem2.cc

```diff
@@ -500,7 +500,9 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
                              f->getName().startswith("verifier.nondet") ||
                              f->getName().startswith("__VERIFIER_nondet")))
         visitNondetCall(CS);
-      else
+      else if (f->getName().startswith("__sea_")) {
```

Comment:
use longer prefix to avoid capturing unintended functions

Reply:
added regex to be more specific.